bin/ovpn_getclient

```diff
@@ -5,25 +5,42 @@
 #
 
 if [ "$DEBUG" == "1" ]; then
-  set -x
+    set -x
 fi
 
 set -e
 
-source "$OPENVPN/ovpn_env.sh"
-cn=$1
+if [ -z "$OPENVPN" ]; then
+    export OPENVPN="$PWD"
+fi
+if ! source "$OPENVPN/ovpn_env.sh"; then
+    echo "Could not source $OPENVPN/ovpn_env.sh."
+    exit 1
+fi
+if [ -z "$EASYRSA_PKI" ]; then
+    export EASYRSA_PKI="$OPENVPN/pki"
+fi
+
+cn="$1"
+parm="$2"
 
 if [ ! -f "$EASYRSA_PKI/private/${cn}.key" ]; then
-    echo "Unable to find ${cn}, please try again or generate the key first"
+    >&2 "Unable to find \"${cn}\", please try again or generate the key first" 1>&2
```

Comment:
This appears to be a typo, did you mean the following? 
`echo "Unable to find \"${cn}\", please try again or generate the key first" 1>&2`